Dockerfile

```diff
@@ -63,12 +62,20 @@ COPY . /app
 RUN yarn build
 
 # =============================
-FROM nginx:1.17 as production
+FROM rhel8/nginx-116 as production
```

Comment:
This is problematic, as it becomes impossible to build the image outside OpenShift. Would be it okay to use explicit URL here? Ie:
registry.redhat.io/rhel8/nginx-116
That would allow building elsewhere as well.

Reply:
Changed this